harvard-stellenbosch-university.csl

```diff
@@ -63,7 +63,7 @@
   </macro>
   <macro name="author-short">
     <names variable="author">
-      <name form="short" and="symbol" delimiter-precedes-last="never" et-al-min="10" et-al-use-first="5" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize-with="." sort-separator=" "/>
+      <name form="short" and="symbol" delimiter-precedes-last="never" et-al-min="10" et-al-use-first="10" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize-with="." sort-separator=" "/>
```

Comment:
Sorry, what are you trying to achieve exactly?

"et-al-use-first" always needs to have a lower value than "et-al-min" (because it doesn't make sense to use et-al-abbreviation but show all authors).